tests/dummy/app/routes/application/template.hbs

```diff
@@ -1,8 +1,8 @@
 <header class="container-fluid">
   <div class="row">
     <h1 class="col-sm-6">
-      <LinkTo @route="index">
-        <img alt="HTML Next Logo" src="./HTML-Next.png" style="height: 2em;">
+      <LinkTo @route="index" style="color:#222">
```

Comment:
why do we end up needing the extra color?

Reply:
The link is an `a` tag and turns blue, but this is the logo on the top left in the header. We want to to be the same color as the logo image, which is `#222`.

This was probably broken in https://github.com/html-next/vertical-collection/commit/3343ecc3506c23c6ba10ed18a3f915672f5ff8c7#diff-a6b32449eac6431ef7263de086a744f8aed3883a686fe93880e607f5e08ffdb9L3, although I don't believe the docs site after that change was ever deployed (it doesn't seem like it, at least).

I agree this is a bit lame, but I think you agree we have bigger fish to fry.